src/main/java/Team/TeamMember.java

```diff
@@ -0,0 +1,61 @@
+package Team;
+
+import equipments.Equipment;
+import equipments.EquipmentStatus;
+import equipments.technology.TechnologyEquipment;
+import equipments.uniform.UniformEquipment;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+public class TeamMember {
+    String name;
+    String teamName;
+    HashMap<String, List<UniformEquipment>> memberUniformEquipment;
+    HashMap<String, List<TechnologyEquipment>> memberTechnologyEquipment;
+
+    public TeamMember(String name, String teamName) {
+        this.name = name;
+        this.teamName = teamName;
+        memberUniformEquipment = new HashMap<String, List<UniformEquipment>>();
+        memberTechnologyEquipment = new HashMap<String, List<TechnologyEquipment>>();
+    }
+
+    public void addUniformEquipment(UniformEquipment equipment) {
+        List<UniformEquipment> newUniformEquipmentType = memberUniformEquipment.get(equipment.equipmentType());
```

Comment:
is `newUniformEquipmentType` an equipment type or a `List`?  this is a confusing name. also, if the equipment's type already exists this isn't new, so the word `new` should not necessarily be used